README.md

```diff
@@ -26,6 +26,15 @@ Read more on the [Confluence space](https://dsdmoj.atlassian.net/wiki/spaces/MRD
 * [Deployment / Helm](./docs/helm-deploy.md) - including how to roll back a deployment
 * [Runbook](./RUNBOOK.md)
 
+### Maintenance Banner
+* To display a maintenance banner to make users aware of any planned outages five environment variables are required. These are available for each environment in the helm_deploy folder -
+  * **SHOW_NOTIFICATION** - this must be set to true for the banner to manifest, by default it is false
```

Comment:
SHOW_NOTIFICATION needs removing from this ReadMe (and the text "five environment variables are required." should be changed to say four.

Reply:
Done!